### PR TITLE
Added SQLite dependency to composer.json

### PR DESCRIPTION
**Pull request by Nex-Otaku**
SQLite extension is required to run demo app, so the extension must be set as dependency.

| Q             | A
| ------------- | ---
| Is bugfix?    | Yes
| New feature?  | No
| Breaks BC?    | No
| Fixed issues  | -

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.4|^8.0",
+        "ext-sqlite3": "^7.4|^8.0",
```
Comment (roxblnfk) [inline composer.json:21]:
```suggestion
        "ext-sqlite3": "*",
```

**Review by roxblnfk — COMMENTED** [review]
It can be more useful

File: composer.json (hunk 1 above)
Comment (roxblnfk) [reply composer.json:21]: Perhaps the pdo + pdo-sqlite should be required instead this

File: composer.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.4|^8.0",
+        "ext-sqlite3": "*",
```
Comment (roxblnfk) [inline composer.json:21]:
```suggestion
        "ext-pdo_sqlite": "*",
```

File: .github/workflows/static.yml [hunk 3]
```diff
@@ -26,6 +26,7 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: "${{ matrix.php }}"
+          extensions: sqlite3
```
Comment (roxblnfk) [inline .github/workflows/static.yml:29]:
```suggestion
```

**Comment by Nex-Otaku** [comment]
Checks failed.. (

**Pull request merged**